test/integration/service/service_test.go

```diff
@@ -161,24 +163,6 @@ func TestGetDevicesForReplay(t *testing.T) {
 	assert.For(ctx, "got").ThatSlice(got).IsNotEmpty()
 }
 
-func TestGetFramebufferAttachment(t *testing.T) {
```

Comment:
We're losing test coverage here -- should we have a test that covers the new mechanism?

Reply:
I went ahead and added a test to TestGet() in the file